src/core/time.d

```diff
@@ -520,7 +520,7 @@ public:
     {
         foreach(T; _TypeTuple!(Duration, const Duration, immutable Duration))
         {
-            foreach(U; _TypeTuple!(Duration, const Duration, immutable Duration))
+            foreach(U; _TypeTuple!(Duration, /*const Duration,*/ immutable Duration))
```

Comment:
I don't know why this doesn't work anymore when compiled with `-cov` :/

Reply:
This is the first build that failed: https://circleci.com/gh/dlang/druntime/1607

So probably this is due to:

https://github.com/dlang/druntime/commit/b38f88f22a264f2df5c7aabb66d1b63834db0ada

(and something else as https://github.com/dlang/druntime/pull/2032 did pass merge being merged)